src/services/fetches.js

```diff
@@ -0,0 +1,17 @@
+
+export const fetchAllBreeds = (page) => {
+    return fetch(`https://api.thedogapi.com/v1/breeds?api_key=f6649eee-08fa-4449-8abe-b57e7641ab6f&limit=15&page=${page}`)
+        .then((res) => res.json());
+};
+
+export const fetchSingleBreed = (id) => {
+    return fetch(`https://api.thedogapi.com/v1/images/search?api_key=f6649eee-08fa-4449-8abe-b57e7641ab6f&breed_id=${id}`)
+        .then((res) => res.json())
+        .then(json => ({
```

Comment:
down below you are using [0] which works. But you could destructor here. (and not use [0]).
```
then(([ json ])=> ({
```